src/taskEvents/script.js

```diff
@@ -213,6 +222,26 @@ function openAddSkillModal() {
   containerDiv.appendChild(skillCategoryDiv);
   containerDiv.appendChild(skillLevelDiv);
   containerDiv.appendChild(submitBtn);
+
+  submitBtn.addEventListener('click', async () => {
+    const skillToAdd = skills?.find(
+      (skill) => skill.name === skillCategorySelect.value,
+    );
+    const levelToAdd = level?.find(
+      (lvl) => lvl.name === skillLevelSelect.value,
+    );
+
+    submitBtn.innerHTML = `<div class="dot-flashing-loader"></div>`;
```

Comment:
not a really good way to do it read [this](https://www.geeksforgeeks.org/what-is-the-disadvantage-of-using-innerhtml-in-javascript/#:~:text=Can%20break%20the%20document%3A%20There,may%20lead%20to%20unexpected%20problems.)

Reply:
fixed